playbooks/save_pubcloud.yml

```diff
@@ -33,6 +33,14 @@
         instance: "{{ openstack_servers[0] }}"
       delegate_to: localhost
 
+    - name: Set the root password to help troubleshooting boot failures
+      user:
+        name: jenkins
+        update_password: always
+        password: "{{ lookup('env', 'IMAGE_JENKINS_PASSWORD') | password_hash('sha512', (65534 | random(seed=inventory_hostname)) | string) }}"
```

Comment:
The salt is done that way to be consistent, although I guess that doesn't really matter for this use-case. Essentially it's a random number between 0 and 65534, but the seed is the inventory_hostname. This makes the seed always consistent, and therefore the result consistent for the server every time it runs... and while every server will have the same password, the hash will be different because of the seed.

Reply:
ok